functions.php

```diff
@@ -169,7 +169,7 @@ function wp_autoupdates_prepare_themes_for_js( $prepared_themes ) {
  * @return bool True if plugins auto-update is enabled, false otherwise.
  */
 function wp_autoupdates_is_plugins_auto_update_enabled() {
-	$enabled = ! defined( 'WP_DISABLE_PLUGINS_AUTO_UPDATE' ) || ! WP_DISABLE_PLUGINS_AUTO_UPDATE;
+	$enabled = true;
```

Comment:
Instead of setting `$enabled` to true, would be cleaner to just pass `true` to the filter.

Reply:
And keep mentioning `$enabled` in the inline docs?

Reply:
yes